examples/src/components/CopyLink.tsx

```diff
@@ -1,33 +1,46 @@
 'use client'
 
 import { useState } from 'react'
-import { LinkIcon } from '@heroicons/react/24/outline'
+import { ArrowUpRightIcon, XMarkIcon } from '@heroicons/react/24/outline'
+
+import Title from '@/components/Title'
 
 export default function CopyLink() {
-  const [copied, setCopied] = useState(false)
+  const [hideCallout, setHideCallout] = useState(false)
 
-  const copyLinkToClipboard = () => {
+  const openLinkInNewTab = async () => {
     const currentPageURL = window.location.href
 
-    navigator.clipboard
-      .writeText(currentPageURL)
-      .then(() => {
-        setCopied(true)
-      })
-      .catch((error) => {
-        console.error('Failed to copy link: ', error)
-      })
+    try {
+      window.open(currentPageURL, '_blank')
+    } catch (err) {
+      console.error('Failed to copy: ', err)
+    }
+  }
+
+  if (hideCallout) {
+    return <></>
   }
 
   return (
-    <div className="pt-6 flex text-left items-center text-neutral-500">
-      <span className="pr-2">Share this document</span>
+    <div className="mr-2 mb-2 text-left items-center text-neutral-500 border-2 border-yellow-200 rounded-lg bg-yellow-50 p-6 ">
+      <div className="flex justify-between pb-2">
+        <Title>How to collaborate on this document</Title>
+        <button onClick={() => setHideCallout(true)}>
+          <XMarkIcon className="h-5 w-5 hover:text-black" />
+        </button>
+      </div>
+      <span className="pr-2">
+        To simulate another user appearing on this document, copy the link to this document and open
+        it in a new window. When you interact on one screen, you should see the user action happens
+        on another.
```

Comment:
Another option:

"To simulate collaborating on this document, open this page in a new window. When you make edits in one window, you should see the document updated in the other."